src/_data/glossary.yml

```diff
@@ -780,6 +780,47 @@
     - "subtyping"
     - "subtype polymorphism"
 
+- term: "Type alias"
+  short_description: |-
+    A user-defined name for an existing type
+  long_description: |-
+    A **type alias** is an alternative name that refers to another type.
+
+    It can be used to simplify complex type definitions, improve readability, or create semantic meaning in code.
+
+    Dart supports type aliases using the `typedef` keyword. You can alias functions, classes, and even generic types.
+
+    ### Examples
+
+    #### Function type alias:
+
+    ```dart
+    typedef StringTransformer = String Function(String);
```

Comment:
Consider adding an example of using this typedef. Perhaps in a function as the type of a a parameter?